src/main/java/training/subsystems/Arm/Elbow/NeoElbow/NeoElbow.java

```diff
@@ -0,0 +1,65 @@
+package training.subsystems.Arm.Elbow.NeoElbow;
+
+import com.revrobotics.CANSparkBase;
+import com.revrobotics.CANSparkLowLevel;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.math.geometry.Rotation2d;
+import training.subsystems.Arm.Elbow.ElbowConstants;
+import training.subsystems.Arm.Elbow.ElbowInputsAutoLogged;
+import training.subsystems.Arm.Elbow.IElbow;
+
+public class NeoElbow implements IElbow {
+
+	private final CANSparkMax motor;
+
+	public NeoElbow() {
+		this.motor = new CANSparkMax(NeoElbowConstants.MOTOR_ID, CANSparkLowLevel.MotorType.kBrushless);
+		motor.getEncoder().setPositionConversionFactor(ElbowConstants.GEAR_RATIO);
+		motor.getEncoder().setVelocityConversionFactor(ElbowConstants.GEAR_RATIO);
+		motor.getPIDController().setP(NeoElbowConstants.P);
+		motor.getPIDController().setI(NeoElbowConstants.I);
+		motor.getPIDController().setD(NeoElbowConstants.D);
+		motor.getEncoder().setPosition(ElbowConstants.PresetPositions.STARTING.ANGLE.getRotations());
```

Comment:
i would put it in logic lair because you would want to di this in each type(real sim)

Reply:
whats logic lair

Reply:
there is the elbow wich in the logic lair.
then there are the talon sim and neo which are physic lair or something like that

Reply:
so i set the pid in the Elbow? but isnt the pid different for the neo and sim?

Reply:
i talk about the 
		motor.getEncoder().setPosition(ElbowConstants.PresetPositions.STARTING.ANGLE.getRotations());

Reply:
ohh k